src/rgw/store/sfs/CHANGELOG.md

```diff
@@ -18,6 +18,11 @@ and this project adheres to
 
 - In order to make stat_bucket call available, SFSBucket::update_container_stat now returns 0.
 
+### Added
+
+- Added mechanism to check for incompatibility isues without changing the
+  original metadata database. Also to warn the user logging a message.
```

Comment:
I suggest to rephrase `Also to warn the user logging a message.`

Reply:
done